packages/core.rb

```diff
@@ -8,7 +8,18 @@ class Core < Package
   compatibility 'all'
 
   is_fake
-
+  depends_on 'gtk3'
+  depends_on 'freetype'
+  depends_on 'cairo'
+  depends_on 'xzutils'
+  depends_on 'libnotify'
+  depends_on 'nspr'
+  depends_on 'libgconf'
+  depends_on 'libxss'
+  depends_on 'libcom_err'
+  depends_on 'gsettings_desktop_schemas'
+  depends_on 'xhost'
+  depends_on 'sommelier'
```

Comment:
Including this in core is going to add a ton of additional dependencies that would require a significant amount of free disk space.  Some users may not want gui apps due to space limitations.